Header header
float64 yaw_ref_true
float64 surge_ref_true
float64 sway_ref_true
float64 alpha_error
float64 beta_error
float64 yaw_error
float64 surge_error
float64 sway_error 
float64 d_yaw_error
float64 tau_u
float64 tau_v
float64 tau_r
float64 s_surge
float64 s_sway
float64 s_yaw
float64 s_yaw_0
float64 tauR_0
float64 tauR_1_bf
float64 tauR_1_af
float64 tauU_1_bf
float64 tauU_1_af
float64 tauV_1_bf
float64 tauV_1_af
float64 param_yaw_eps
float64 param_yaw_kc
float64 param_yaw_lambda
float64 param_surge_eps
float64 param_surge_kc
float64 param_surge_lambda
float64 param_sway_eps
float64 param_sway_kc
float64 param_sway_lambda